compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -212,8 +214,7 @@ namespace jank::runtime
      * moves the `llvm::Module` held in the `PartialTranslationUnit`. */
     if(truthy(compile_files_var->deref()))
     {
-      auto module_name{ runtime::to_string(current_module_var->deref()) };
```

Comment:
So, aside from moving the `set_is_loaded` and `is_loaded` to RT context, which isn't a behavior change, this is actually the logical change.

We have a var to track the current module. Why is it not sufficient? You're using a parameter instead. This is suspicious to me, since we'll have other places in the code relying on the `current_module_var` holding the current module. Why are we not setting it in the loader when load `.cpp` modules. In fact, *where are we setting it*?

These are questions we need to understand, answer, and (if we're keeping this code) ultimately document. My gut tells me there's no reason we can't use the var for this.